docs/public/content/demo/demo_guide.md

```diff
@@ -0,0 +1,134 @@
++++
+title = "mFlow Demo"
+insert_anchor_links = "right"
+weight = 1
++++
+
+## Introduction
+
+Demo can be accessed at: [https://irims-demo.msupply.org:50000/](https://irims-demo.msupply.org:50000/)
+
+This demo allows you to use the **mFlow Application Manager** system to experience the following workflow:
+
+1. **Create a user account**
+
+   This is an automated process that requires email verification.
+
+2. **Register a company**
+
+   A reviewer will approve this application, which will create your organisation in the system.
+
+3. **Apply for a license for your company to register products**
+
+   This will require 2 levels of review -- an initial screening, and an an assessment by a more senior reviewer. You will receive a PDF license and permission to register a product (for 3 days)
```

Comment:
```suggestion
   This will require 2 levels of review -- an initial screening, and an assessment by a more senior reviewer. You will receive a PDF license and permission to register a product (for 3 days)
```

Reply:
@nmadruga -- FYI I just arbitrarily put 3 days in here. Does that sound about right for the demo? (If so, let's update the demo template)